src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-event-pool-size.mdx

```diff
@@ -0,0 +1,565 @@
+---
+title: Set max event pool size
+type: apiDoc
+shortDescription: 'Sets the maximum size of the event pool.'
+tags:
+  - Mobile monitoring
+  - Mobile SDK API
+  - Custom instrumentation
+metaDescription: Mobile app monitoring API used to set the maximum size of the event pool.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setmaxeventpoolsize-android-agent-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setmaxeventpoolsize-android-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-max-event-pool-size
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/setmaxeventpoolsize-ios-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setmaxeventpoolsize-android-agent-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setmaxeventpoolsize-android-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-pool-size
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+        <TabsBarItem id="capacitor">
+            Capacitor
+        </TabsBarItem>
+        <TabsBarItem id="cordova">
+            Cordova
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React Native
+        </TabsBarItem>
+        <TabsBarItem id="xamarin">
+            Xamarin
+        </TabsBarItem>
+        <TabsBarItem id="maui">
+            .NET MAUI
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java]
+
+```java
+NewRelic.setMaxEventPoolSize(int $maxSize)
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
+
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
```suggestion
```kotlin
NewRelic.setMaxEventPoolSize(maxSize: Int)
```

```